src/Symfony/Component/Routing/Tests/Fixtures/dumper/url_matcher5.php

```diff
@@ -195,9 +165,43 @@ public function match($rawPathinfo)
                 }
 
                 return $ret;
+                not_slashed_c:
+                break;
+        }
+
+        $regexOffset = 0;
+        $regexList = array(
```

Comment:
Not sure if it's useful in this case, but the `S` modifier makes PHP better analyze the regexps to improve performance: http://php.net/manual/en/reference.pcre.pattern.modifiers.php

> S
>
> When a pattern is going to be used several times, it is worth spending more time analyzing it in order to speed up the time taken for matching. If this modifier is set, then this extra analysis is performed. At present, studying a pattern is useful only for non-anchored patterns that do not have a single fixed starting character.

Reply:
> studying a pattern is useful only for non-anchored patterns

the pattern here is anchored, so this doesn't apply